resources/nl/cemu.po

```diff
@@ -984,929 +1181,1278 @@ msgstr ""
+msgid "&NN NFP"
+msgstr "&NN NFP"
+
+#: MainWindow.cpp
 msgid "&GX2 API"
 msgstr "&GX2 API"
 
-#: MainWindow.cpp:2368
+#: MainWindow.cpp
 msgid "&Audio API"
 msgstr "&Audio API"
 
-#: MainWindow.cpp:2369
+#: MainWindow.cpp
 msgid "&Input API"
 msgstr "&Input API"
 
-#: MainWindow.cpp:2370
+#: MainWindow.cpp
 msgid "&Socket API"
 msgstr "&Socket API"
 
-#: MainWindow.cpp:2371
+#: MainWindow.cpp
 msgid "&Save API"
 msgstr "&Save API"
 
-#: MainWindow.cpp:2372
+#: MainWindow.cpp
 msgid "&H264 API"
 msgstr "&H264 API"
 
-#: MainWindow.cpp:2374
+#: MainWindow.cpp
+msgid "&Graphic pack patches"
+msgstr "&Graphic pack patches"
+
+#: MainWindow.cpp
 msgid "&Texture cache warnings"
 msgstr "&Texture cache waarschuwingen"
 
-#: MainWindow.cpp:2376
+#: MainWindow.cpp
 msgid "&OpenGL debug output"
 msgstr "&OpenGL debug uitvoer"
 
-#: MainWindow.cpp:2377
-msgid "&Vulkan validation layer"
-msgstr "&Vulkan validatie laag"
+#: MainWindow.cpp
+msgid "&Vulkan validation layer (slow)"
+msgstr "&Vulkan validatie laag (traag)"
 
-#: MainWindow.cpp:2381
+#: MainWindow.cpp
+msgid "&Log PPC context for API"
+msgstr "&Log PPC context voor API"
+
+#: MainWindow.cpp
 msgid "&Textures"
 msgstr "&Textures"
 
-#: MainWindow.cpp:2382
+#: MainWindow.cpp
 msgid "&Shaders"
 msgstr "&Shaders"
 
-#: MainWindow.cpp:2383
+#: MainWindow.cpp
 msgid "&nlibcurl HTTP/HTTPS requests"
 msgstr "&nlibcurl HTTP/HTTPS verzoeken"
 
-#: MainWindow.cpp:2387
+#: MainWindow.cpp
 msgid "&Logging"
 msgstr "&Logging"
 
-#: MainWindow.cpp:2388
+#: MainWindow.cpp
 msgid "&Dump"
 msgstr "&Dump"
 
-#: MainWindow.cpp:2392 MainWindow.cpp:2397
+#: MainWindow.cpp
 msgid "&Render upside-down"
 msgstr "&Geef op zijn kop weer"
 
-#: MainWindow.cpp:2401
+#: MainWindow.cpp
+msgid "&Async compile (Vulkan)"
+msgstr "&Async compilatie (Vulkan)"
+
+#: MainWindow.cpp
+msgid "&Experimental"
+msgstr "&Experimenteel"
+
+#: MainWindow.cpp
 msgid "&Open logging window"
 msgstr "&Open log scherm"
 
-#: MainWindow.cpp:2403
+#: MainWindow.cpp
 msgid "&View PPC threads"
 msgstr "&Open PPC threads"
 
-#: MainWindow.cpp:2404
+#: MainWindow.cpp
 msgid "&View PPC debugger"
 msgstr "&Open PPC debugger"
 
-#: MainWindow.cpp:2405
+#: MainWindow.cpp
 msgid "&View audio debugger"
 msgstr "&Open audio debugger"
 
-#: MainWindow.cpp:2406
+#: MainWindow.cpp
 msgid "&View texture cache info"
 msgstr "&Open texture cache informatie"
 
-#: MainWindow.cpp:2407
+#: MainWindow.cpp
 msgid "&Show frame profiler"
 msgstr "&Open frame profiler"
 
-#: MainWindow.cpp:2408
+#: MainWindow.cpp
 msgid "&Dump current RAM"
 msgstr "&Dump huidige RAM"
 
-#: MainWindow.cpp:2409
+#: MainWindow.cpp
 msgid "&Dump WUD filesystem"
 msgstr "&Dump WUD bestandssysteem"
 
-#: MainWindow.cpp:2411
+#: MainWindow.cpp
 msgid "&Debug"
 msgstr "&Debug"
 
-#: MainWindow.cpp:2416
+#: MainWindow.cpp
 msgid "&Check for updates"
 msgstr "&Controleer voor updates"
 
-#: MainWindow.cpp:2417
+# Couldn't really find a great translation for this word unfortunately.
```

Comment:
That's fine! [Microsoft Terminology database](https://www.microsoft.com/en-us/language/Search?&searchTerm=Getting%20started&langID=155&Source=true&productid=undefined) agrees with your translation.
Remove this line the next time you edit the file.

Reply:
That's a handy website, I'll definitely be bookmarking that!